mmv1/products/identityplatform/Config.yaml

```diff
@@ -254,3 +254,116 @@ properties:
+  - !ruby/object:Api::Type::NestedObject
+    name: 'mfa'
+    description: |
+      Options related to how clients making requests on behalf of a project should be configured.
+    ignore_read: true
+    properties:
+      - !ruby/object:Api::Type::Enum
+        name: 'state'
+        description: |
+          Whether MultiFactor Authentication has been enabled for this project.
+        values:
+          - :STATE_UNSPECIFIED
+          - :DISABLED
+          - :ENABLED
+          - :MANDATORY
+      - !ruby/object:Api::Type::Array
+        name: enabledProviders
+        description: |
+          A list of usable second factors for this project.
+        item_type: !ruby/object:Api::Type::Enum
+          name: 'undefined'
+          description: |
+            This field only has a name and description because of MM
+            limitations. It should not appear in downstreams.
+          values:
+            - :PROVIDER_UNSPECIFIED
+            - :PHONE_SMS
+      - !ruby/object:Api::Type::NestedObject
+        name: providerConfigs
+        description: |
+          A list of usable second factors for this project along with their configurations.
+          This field does not support phone based MFA, for that use the 'enabledProviders' field.
+        properties:
+          - !ruby/object:Api::Type::Enum
+            name: 'state'
+            description: |
+              Whether MultiFactor Authentication has been enabled for this project.
+            values:
+              - :STATE_UNSPECIFIED
+              - :DISABLED
+              - :ENABLED
+              - :MANDATORY
+          - !ruby/object:Api::Type::NestedObject
+            name: totpProviderConfig
+            description: |
+              TOTP MFA provider config for this project.
+            properties:
+              - !ruby/object:Api::Type::Integer
+                name: adjacentIntervals
+                description: |
+                  The allowed number of adjacent intervals that will be used for verification to avoid clock skew.
+  - !ruby/object:Api::Type::NestedObject
+    name: 'multiTenant'
+    description: |
+      Configuration related to multi-tenant functionality.
+    properties:
+      - !ruby/object:Api::Type::Boolean
+        name: 'allowTenants'
+        description: |
+          Whether this project can have tenants or not.
+      - !ruby/object:Api::Type::String
+        name: 'defaultTenantLocation'
+        description: |
+          The default cloud parent org or folder that the tenant project should be created under.
+          The parent resource name should be in the format of "/", such as "folders/123" or "organizations/456".
+          If the value is not set, the tenant will be created under the same organization or folder as the agent project.
+  - !ruby/object:Api::Type::NestedObject
+    name: 'monitoring'
+    description: |
+      Configuration related to monitoring project activity.
+    ignore_read: true
```

Comment:
I wonder what is the reason to add ignore_read ?

Reply:
Same reason as in https://github.com/GoogleCloudPlatform/magic-modules/pull/9609#discussion_r1428047312

Reply:
`default_from_api` should be used here instead of `ignore_read`